examples/cpchop.cpp

```diff
@@ -75,6 +76,7 @@ try
     Opm::ParameterGroup param(argc, argv);
     std::string gridfilename = param.get<std::string>("gridfilename");
     Opm::CornerPointChopper ch(gridfilename);
+    Opm::Parser parser;
```

Comment:
i think it would make sense if this was a chopper internal.

Reply:
OK - fixed